experiment/popularity_community/crawl_torrents.py

```diff
@@ -0,0 +1,214 @@
+"""This script crawl first 100 torrens from random nodes in the network.
+
+### Usage
+
+```
+export PYTHONPATH=${PYTHONPATH}:`echo ../../../src/{pyipv8,tribler-common,tribler-core} | tr " " :`
+
+python3 crawl_torrents.py [-t <timeout_in_sec>] [-f <output_file.json>]
+```
+
+Where:
+* `timeout_in_sec` means a time that the experiment will last
+* `output_file.json` means a path and a result file name
+
+### Example
+
+```
+python3 crawl_torrents.py -t 600
+python3 crawl_torrents.py -t 600 -f result.json
+```
+
+"""
+import argparse
+import asyncio
+import json
+import logging
+import os
+from collections import defaultdict
+
+import lz4
+
+from ipv8.lazy_community import lazy_wrapper
+from ipv8.peer import Peer
+from ipv8.peerdiscovery.discovery import RandomWalk
+from tool.tiny_tribler_service import TinyTriblerService
+from tribler_core.modules.metadata_store.community.remote_query_community import RemoteQueryCommunity, \
+    SelectResponsePayload, SelectRequest, RemoteSelectPayload
+from tribler_core.modules.metadata_store.serialization import REGULAR_TORRENT
+from tribler_core.utilities.unicode import hexlify
+
+UNLIMITED = -1  # this magic number is use inside many Tribler modules
+
+IPV8_WALK_INTERVAL = 0.05
+
+RANDOM_WALK_TIMEOUT_IN_SEC = 5
+RANDOM_WALK_WINDOW_SIZE = 0
+RANDOM_WALK_RESET_CHANCE = 30
+
+CRAWLER_REQUEST_TIMEOUT_IN_SEC = 60
```

Comment:
:+1:  for `_IN_SEC` thing. I should start doing that myself...

Reply:
Good one. We have tons of timers/timeouts which can be enhanced by adding the unit to the variable name.

Reply:
Probably it is better to have all timeouts at the same scale (seconds), to simplify arithmetic on it. In that case, it is not necessary to specify a unit as part of the name. And longer timeout may be specified as
```python
SOME_LONG_TIMEOUT = 10 * HOUR  #  where HOUR = 60 * MINUTE, MINUTE = 60 * SECOND, and SECOND = 1
```

Reply:
I also like that idea. I think all of our timeouts are in seconds already. 👍

Reply:
Maybe I came from "strange dev-place", but I have seen that different modules uses different defaults for the time.
(seconds and microseconds, basically)

So, every time, i see "delay" or "interval" or "time", I can't realise, what exactly this means. 

Therefore, I prefer explicit naming.

Reply:
I propose to make a convention that in Tribler all time-related values should be in *seconds*. When we work with some external library we should add a time scale prefix, like @drew2a did.
What do you think, guys?

Reply:
I support using seconds as the default, as it is the grounding for all time in Python (stemming from `time.time()`).